packages/de1/src/converters/shot.ts

```diff
@@ -40,6 +38,11 @@ function decode(data: Buffer): Shot {
   return { ...vars, time };
 }
 
+function convert_3_char_to_U24P16(buffer: Buffer): number {
+  const value = buffer[0] + buffer[1] / 256 + buffer[2] / 65536;
+  return value;
```

Comment:
```ts
return buffer[0] + buffer[1] / 256 + buffer[2] / 65536;
```

?

Reply:
Yes I’d prefer even shorter, dropping the `{}` and `return`, but not sure if that fits in code style?

Reply:
Yes I’d prefer even shorter, dropping the {} and return, but not sure if that fits in code style?

Reply:
In this case I wouldn't do it because yes, the other files are in the other style. And I believe I wrote them in that styles lest to mix `function foo() {}` with `const bar = () => …` in the same file